_posts/2020-12-11-trans-positive.markdown

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: "Normal compliments for trans people"
+date: 2020-12-11 01:00:00
+categories: trans
+---
+
+{::options parse_block_html="true" /}
+
+# The short version
+{: #short}
+
+{% include collapser.markdown %}
+
+<div>
+
+As a trans person, you hear various compliments, and you might think,
+*they're probably just saying it to make me feel better, but they don't really mean it.
+Look at me, this disgusting thing, what is likeable about it?
+They say nice things, but there's no way they believe it.*
+
+Then, to you, I would ask, why don't you believe it?
+
+In a world where trans people are accepted and being trans is considered normal,
+these compliments are not strange at all.
+
+Indeed, trans people can be beautiful, attractive, desired, hot, and other good things.
+Well, those traits in particular may not be good to you, but you can have your good traits that you want!
+
+Rather, that I was suspicious, that I thought others were lying,
+says less about their credibility and more about how much transphobia I've internalized.
+After being constantly told I'm worthless and unwanted, of course these compliments would be hard to believe.
+It's not just that I didn't think I could be seen positively -
+I thought trans people in general were only seen as something lesser.
+Kind words directed at others, I rejected in the same way.
+
+This baggage can be unlearned. Not overnight, but with time.
+When people say good things about you, they can and probably do mean it.
+Let's show our trans siblings some love, and help them become their best self.
+
+</div>
+
+# The long version
+{: #long}
+
+{% include collapser.markdown %}
+
+<div>
+
+I am transfem, as are my girlfriends, and I preferentially hang around transfem spaces if I interact with the outside world at all, so I suppose it would be my domain of expertise.
+Common compliments include cute, pretty, and beautiful, and we tend to consider these to be good things.
+Not everyone gets dysphoria, but I do get it, and I can find it hard to see myself as beautiful.
+Being on HRT now, being free to explore without unwelcome influence,
```

Comment:
Perhaps include the full name in brackets too (some people reading the post might not know what HRT stands for, although I guess they could also search up the acronym)

Reply:
Searching `hrt` gives hormone replacement therapy as the second result... but it's referring to a different kind of hormone replacement therapy. Searching `hormone replacement therapy` gives the same result.

Searching `trans hrt` quickly gets the correct result.

I don't think there's much benefit to putting the full words.